src/components/AvatarWithImagePicker.js

```diff
@@ -116,166 +116,118 @@ const defaultProps = {
     originalFileName: '',
 };
 
-class AvatarWithImagePicker extends React.Component {
-    constructor(props) {
-        super(props);
-        this.animation = new SpinningIndicatorAnimation();
-        this.setError = this.setError.bind(this);
-        this.isValidSize = this.isValidSize.bind(this);
-        this.showAvatarCropModal = this.showAvatarCropModal.bind(this);
-        this.hideAvatarCropModal = this.hideAvatarCropModal.bind(this);
-        this.state = {
-            isMenuVisible: false,
-            validationError: null,
-            phraseParam: {},
-            isAvatarCropModalOpen: false,
-            imageName: '',
-            imageUri: '',
-            imageType: '',
+function AvatarWithImagePicker(props) {
+    const animation = new SpinningIndicatorAnimation();
```

Comment:
this will be recreated on every new frame, I think is better to use `useRef`
```
    const animation = useRef(new SpinningIndicatorAnimation()).current;
```